extensions/Worker.Extensions.Storage.Blobs/src/Config/BlobStorageBindingOptions.cs

```diff
@@ -17,7 +17,7 @@ internal class BlobStorageBindingOptions
 
         public BlobClientOptions? BlobClientOptions { get; set; }
 
-        public BlobServiceClient CreateClient()
+        internal virtual BlobServiceClient CreateClient()
```

Comment:
should this options object cache the client? Can then also use that to set the client during a unit test instead of mocking.

```suggestion
        internal BlobServiceClient? Client { get; set; }

        internal BlobServiceClient CreateClient()
        {
            if (this.Client is not null)
            {
                return this.Client;
            }
            
            // create client logic
        }
```

Reply:
Yup! I think there was an issue to do this but will fix here since a) needs to be done and b) cleans up the  unit test!